Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN cd /root && \
 COPY pywps.wsgi /var/www/html/wps/
 COPY apache2.conf /etc/apache2/
 COPY pywps.cfg /etc/
-COPY wps_processes/wps_*.py /var/www/html/wps/
+COPY pavics_datacatalog/wps_processes/wps_*.py /var/www/html/wps/
```

Comment:
Here we could copy the whole repo and then install it rather than just copying the processes.
I don't thing that the path /var/www/html/wps/ is still relevant